src/App.css

```
@@ -1,42 +1,1836 @@
+    gap: 16px;
+    transition: all 0.3s ease;
+    color: var(--text-primary);
+    font-family: inherit;
+  }
+  
+  .qa-question:hover {
+    background: var(--bg-tertiary);
+  }
+  
+  .qa-item.active .qa-question {
+    background: linear-gradient(135deg, var(--primary-color)0a, var(--primary-light)0a);
```

Stylelint (17.14.0)

[error] Unknown unit "a" (unit-no-unknown)

(unit-no-unknown)